packages/rocketchat-ui-sidenav/client/sideNav.js

```diff
@@ -85,3 +104,11 @@ Template.sideNav.onRendered(function() {
 
 	return Meteor.defer(() => menu.updateUnreadBars());
 });
+
+Template.sideNav.onCreated(function() {
+	this.autorun(() => {
+		if (!Meteor.user().settings.preferences.collapseChannels) {
```

Comment:
@geekgonecrazy I only saw this approach with the user preferences. Can we use Rocketchat.settings to get the user preferences?